server/index.js

```diff
@@ -1,13 +1,23 @@
 var express = require('express');
 var parser = require('body-parser');
 var db = require('../db/index.js');
+var path = require('path');
 
 let app = express();
 app.use(parser.json());
 app.use(express.static(__dirname + '/../react-client/dist'));
 
+app.get('/:id', (req, res) => {
+  res.sendFile(path.resolve('react-client/dist/index.html'));
+})
+
 app.get('/item/:id', (req, res) => {
-  console.log(req);
+  let id = req.params.id;
+  q = `select items.name, items.price from items where items.item_id = ${id}`

```

Comment:
could this query call be moved into the model?  could be more readable if additional logic comes into play here later.  'Move query logic into model' would be a nice, quick ticket

Reply:
Good idea! I will look to implement.